NetCoreRestApi/DataLayer/UnitOfWorks/ProductUnitOfWork.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DataLayer.Entities;
+using DataLayer.Repositories;
+using DataLayer.UnitOfWorks.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DataLayer.UnitOfWorks
+{
+    public class ProductUnitOfWork<TId> : IProductUnitOfWork<TId>
```

Comment:
Fix to class ProductUnitOfWork : BaseUnitOfWork<ProdcutModel, int>, IProductUnitOfWork

Reply:
done